include/dem/pw_contact_force.h

```diff
@@ -158,7 +179,10 @@ class PWContactForce
   std::map<types::particle_index, double> effective_coefficient_of_restitution;
   std::map<types::particle_index, double> effective_coefficient_of_friction;
   std::map<types::particle_index, double>
-    effective_coefficient_of_rolling_friction;
+                                effective_coefficient_of_rolling_friction;
+  std::map<int, Tensor<1, dim>> ForceOnWall;
+  std::map<int, Tensor<1, dim>> TorqueOnWall;
+  bool                          calculation_force_torque;
```

Comment:
```suggestion
  std::map<int, Tensor<1, dim>> force_on_walls;
  std::map<int, Tensor<1, dim>> torque_on_wall;
  bool                          calculate_force_torque_on_boundary;
```
Be more verbose in your variable names.

Reply:
Yes